java/src/main/native/src/TableJni.cpp

```diff
@@ -935,6 +935,39 @@ jlongArray combine_join_results(JNIEnv *env, cudf::table &left_results,
   return combine_join_results(env, std::move(left_cols), std::move(right_cols));
 }
 
+cudf::column_view remove_validity_from_col(cudf::column_view column_view) {
+  if (column_view.nullable() && column_view.null_count() == 0) {
+    // null_mask is allocated but no nulls present therefore we create a new column_view without the
+    // null_mask to avoid things blowing up in reading the parquet file
+    if (!cudf::is_nested(column_view.type())) {
+      cudf::column_view cv(column_view.type(), column_view.size(), column_view.data<int32_t>(),
+                           nullptr, 0);
+      return cv;
```

Comment:
It appears this will only work with INT32 columns. 
Also, sliced column's will have an offset which should be passed on the ctor as well.

Reply:
Like David said, if you want to support generic type, use `void` type.
```suggestion
      return cudf::column_view(column_view.type(), column_view.size(), column_view.template data<void>(), nullptr, 0, column_view.offset());
```

Reply:
I am now using `head()` which is better I think because we don't have to worry about `offset`